lib/mongo/retryable.rb

```diff
@@ -123,7 +123,7 @@ def write_with_retry(session, write_concern, &block)
 
     def retry_write_allowed?(session, write_concern)
       session && session.retry_writes? &&
-          (write_concern.nil? || write_concern.acknowledged?)
+          (write_concern.nil? || write_concern.acknowledged?) or false
```

Comment:
Just to make sure, is the point of using `or false` here to ensure that the method returns false instead of nil?

Reply:
Yep.